File: Selenium/Project/FST_HRM_Project/src/test/java/Activity/Activity_03.java
package Activity;
import org.testng.Assert;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;



public class Activity_03 {
    public static void main(String[] args) throws InterruptedException {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("http://alchemy.hguy.co/orangehrm");
        driver.findElement(By.xpath("//input[@name='txtUsername']")).sendKeys("orange");
        driver.findElement(By.xpath("//input[@id='txtPassword']")).sendKeys("orangepassword123");
        driver.findElement(By.id("btnLogin")).click();
        Thread.sleep(3000);
        String HomePageTitle=driver.getTitle();
        String ExpectedHometitle="OrangeHRM";
        System.out.println(HomePageTitle);
        Assert.assertEquals(HomePageTitle,ExpectedHometitle);
        driver.close();

    }
}

File: Python/Activities/Project/Activity-03.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from selenium.webdriver.support.wait import WebDriverWait
from webdriver_manager.firefox import GeckoDriverManager


service = FirefoxService(GeckoDriverManager().install())

# start the driver
with webdriver.Firefox(service=service) as driver:
    driver.get("http://alchemy.hguy.co/orangehrm")
    driver.find_element(By.XPATH,"//input[@name='txtUsername']").send_keys("orange")
    driver.find_element(By.XPATH,"//input[@id='txtPassword']").send_keys("orangepassword123")
    driver.find_element(By.ID,"btnLogin").click()
    wait = WebDriverWait(driver, 5)
    Homepage=driver.title
    print(Homepage)
    assert Homepage,"OrangeHRM"
    driver.close()